src/js/api/config.js

```diff
@@ -47,10 +47,11 @@ define([
             nextUp: 'Next Up',
             nextUpClose: 'Next Up Close',
             related: 'Discover',
-            close: 'Close',
+            close: 'Close'
         },
         renderCaptionsNatively: true,
-        nextUpDisplay: true
+        nextUpDisplay: true,
+        preload: 'metadata'
```

Comment:
Let's not add a player default. Set the default on the item where this would be applied if no item preload setting was made explicitly.